### PR TITLE
Grovekit update

### PR DESCRIPTION
**Pull request by daisukeiot**
## Purpose

* Update documentations to match user experience with customized Raspbian.
* change Mic hat name from 2 mic to 4 mic.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code

Run scenarios 3

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by daisukeiot** [comment]
Documentation updates and 2Mic to 4 Mic change

File: seeed/1-temperature-sensor/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You will need the following hardware to complete this project:
 
 - A laptop (**Note**: The instructions in this document have been tested on Mac and Windows.)
 - The Grove Starter Kit for Azure IoT Edge
-- A Raspberry Pi 3
+- A 5V Power Supplly with Micro USB interface
```
Comment (jessebenson) [inline seeed/1-temperature-sensor/README.md]: (sp):  "Supply"

File: seeed/2-image-classifier/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ You will need the following hardware to complete this project:
 
 - A laptop (**Note**: The instructions in this document have been tested on Mac and Windows.)
 - The Grove Starter Kit for Azure IoT Edge
-- A Raspberry Pi 3
+- A 5V Power Supplly with Micro USB interface
```
Comment (jessebenson) [inline seeed/2-image-classifier/README.md]: (sp):  Supply

File: seeed/2-image-classifier/README.md [hunk 3]
```diff
@@ -26,25 +26,24 @@ In this project, your Raspberry Pi 3 will be configured as an Azure IoT Edge dev
 
 - Micro SD card
 - Keyboard and monitor
-- Camera
-- ReSpeaker 2-Mics Pi HAT
-- OLED display
+- Raspberry Pi Camera Module V2
+- ReSpeaker 4-Mic Array for Raspberry Pi
+- Grove - OLED Display 0.96''
```
Comment (jessebenson) [inline seeed/2-image-classifier/README.md]: (nit):  double quotes instead of two single quotes

**Pull request merged**